assignments/function-conversion.js

```diff
@@ -1,30 +1,23 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
-// console.log("Function was invoked!");
-// };
-// myFunction();
-
-// let anotherFunction = function (param) {
-//   return param;
-// };
-// anotherFunction("Example");
-
-// let add = function (param1, param2) {
-//   return param1 + param2;
-// };
-// add(1,2);
-
-// let subtract = function (param1, param2) {
-//   return param1 - param2;
-// };
-// subtract(1,2);
-
-
-// Stretch
-
-// exampleArray = [1,2,3,4];
-// const triple = exampleArray.map(function (num) {
-//   return num * 3;
-// });
-// console.log(triple);
\ No newline at end of file
+const myFunction = ()=>"Function was invoked!";
+console.log(myFunction())
+
+const anotherFunction = (param)=> param;
```

Comment:
const anotherFunction = param => param;

when there's only a single parameter you don't need parenthesis